packages/common/composable/http/axios.js

```
@@ -0,0 +1,151 @@
+
+  if (typeof MockAdapter.prototype.proxy === 'undefined') {
+    MockAdapter.prototype.proxy = function ({ url, config = {}, proxy, response, handleData } = {}) {
+      let stream = this
+      const request = (proxy, any) => {
+        return (setting) => {
+          return new Promise((resolve) => {
+            config.responseType = 'json'
+            axios
+              .get(any ? proxy + setting.url + '.json' : proxy, config)
+              .then(({ data }) => {
+                typeof handleData === 'function' && (data = handleData(data, setting))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)